src/Orchard.Cms.Web/Modules/Orchard.Demo/Views/Content/Display.cshtml

```diff
@@ -7,5 +7,5 @@
     var displayText = ContentManager.PopulateAspect<ContentItemMetadata>(contentItem).DisplayText;
 }
 
-<h1>@RenderTitleSegments(displayText)</h1>
+<h1>@RenderTitleSegments(T[displayText])</h1>
```

Comment:
Just because it is the only view which fails when precompiling. Not sure this view is currently used, otherwise what is the correct fix?